store/todos.js

```diff
@@ -124,4 +149,36 @@ export const mutations = {
     state.perPage = 9;
     state.page = 1;
   },
+  setListPerPage(state, val) {
+    state.taskListPerPage = state.taskList;
+  },
+  filterTaskList(state, val) {
+    state.filterOptions.forEach((element) => (element.status = false));
+    const option = state.filterOptions.find((option) => option.id === val.id);
+
+    option.status = true;
+
+    // eslint-disable-next-line camelcase
+    if (val.title === ALL_TASK) {
+      state.taskListPerPage = state.taskList;
+
+      return;
+    }
+
+    // eslint-disable-next-line camelcase
+    if (val.title === INCOMPLETE_TASK) {
+      state.taskListPerPage = state.taskList.filter(
+        (task) => task.done === false
+      );
+
+      return;
+    }
+
+    // eslint-disable-next-line camelcase
+    if (val.title === COMPLETE_TASK) {
```

Comment:
why these rules are ignored?

Reply:
I forgot to remove the commented section after fixing the constant name to all uppercase